packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -134,8 +135,18 @@ exports.list = async (req, res) => {
 
 			// default question format
 			const key = question.title ?? question.question;
-			const value =
+			const currentQuestion =
 				question.altText ?? journey.response?.answers[question.fieldName] ?? 'Not started';
+			let value = '';
+
+			if (currentQuestion.uploadedFiles) {
+				for (const item in currentQuestion.uploadedFiles) {
```

Comment:
currently displaying the files as a plain text list, future ticket in the works to update this to downloadable links